commit dd18e9a13558cbe519c382d3fe358ae97e39cd38
Author: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date:   Thu Mar 17 15:32:21 2016 +0100

    filetemplates plugin: Make testfiletemplates depend on kdevfiletemplates

    There is currently no dependency between both targets, however some of
    the files built by both testfiletemplates and kdevfiletemplates (such as
    classidentifierpage.cpp) depend on ui files that only the latter target
    generates. This means that if one does make testfiletemplates the build
    fails like this:

      /tmp/kdevplatform/plugins/filetemplates/classidentifierpage.cpp:23:25:
      fatal error: ui_newclass.h: No such file or directory

    Differential Revision:	https://phabricator.kde.org/D1160

--- plugins/filetemplates/CMakeLists.txt
+++ plugins/filetemplates/CMakeLists.txt
@@ -81,6 +81,7 @@ main.cpp
 )
 
 kde4_add_executable(testfiletemplates ${test_srcs})
+add_dependencies(testfiletemplates kdevfiletemplates)
 
 target_link_libraries(testfiletemplates
     ${KDE4_KDECORE_LIBS}
